src/diagnose.ts

```diff
@@ -205,7 +205,9 @@ export class DiagnoseTool {
 
     Object.keys(options).forEach(key => {
       const newKey = JS_TO_RUBY_MAPPING[key]
-      config[newKey] = options[key]
+      if (newKey !== undefined) {
```

Comment:
I guess this is because the OpenTelemetry config options aren't in our JS_TO_RUBY_MAPPING list?